path.go

```diff
@@ -49,18 +51,29 @@ func zoneFromPath(host string, path string, rec record) (string, int, error) {
 	return strings.Join(url, "."), from, nil
 }
 
-func getFinalRecord(zone string, from int) (record, error) {
+func getFinalRecord(zone string, from int, ctx context.Context, c Config) (record, error) {
 	var txts []string
 	var err error
 
 	txts, err = net.LookupTXT(zone)
```

Comment:
Early feedback: the 'standard' lookup should only happen if the user didn't provide a custom resolver